Nodejs/Product/Nodejs/Options/NodejsIntellisenseOptionsPage.cs

```diff
@@ -25,20 +25,7 @@
 namespace Microsoft.NodejsTools.Options {
     [ComVisible(true)]
     public class NodejsIntellisenseOptionsPage : NodejsDialogPage {
-        private static readonly Version _typeScriptMinVersionForES6Preview = new Version("1.6");
```

Comment:
So this tells me that we are *only* targeting Dev15 from here on out. Is that correct? If so, we need to balance that with the ability to get feedback on dev builds, so we should block in the installer, or provide a link to install the latest TS bits if it doesn't exist on startup